cdci_data_analysis/analysis/parameters.py

```diff
@@ -297,6 +274,10 @@ def set_par(self, value, units=None):
         if units is not None:
             self.units = units
         self.value = value
+        # this should return the value in a default format
+        return self.get_value_default_format(value)
+
+    def get_value_default_format(self, value):
```

Comment:
maybe let's call it something like `get_value_in_default_format`?
Else it may sound like "get default format of value" or something.